core/src/migrations/000074-mergeCompleteImportTask.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  up: async function (migration) {
+    await migration.sequelize.transaction(async () => {
+      // Kick profiles back to "pending" if they're ready but their imports are not
+      await migration.sequelize.query(
+        `UPDATE profiles SET "state"='pending' WHERE "state"='ready' AND "id" IN 
+          (SELECT "profileId" FROM imports WHERE "exportedAt" IS NULL AND "profileUpdatedAt" IS NULL AND "groupsUpdatedAt" IS NULL)`
```

Comment:
```sql
UPDATE "profiles" SET "state"='pending' WHERE "state"='ready' AND "id" IN 
          (SELECT DISTINCT("profileId") FROM "imports" WHERE "exportedAt" IS NULL AND "profileUpdatedAt" IS NULL AND "groupsUpdatedAt" IS NULL)
```

* add quotes around table names
* added distinct around inner select

Reply:
Confirming that after this PR, we shouldn't be able to get into this state again